learn/experimental/overview.mdx

```diff
@@ -0,0 +1,44 @@
+# Experimental features
+
+Meilisearch periodically introduces new experimental features. Experimental features are not always ready for production, but offer functionality that might benefit some users.
```

Comment:
I was discussing this with @dureuill in a separate conversation. In short, while it is true some experimental features are mostly ready and only marked as experimental because of possible API changes (e.g. metrics), that is not the case for others. For example, my understanding is that `--experimental-reduce-indexing-memory-usage` is experimental not because its API might change, but because we want to see it in the wild before fully integrating it into our codebase. @dureuill mentioned vector search implementation is also not as strong and thorough as a "regular" feature.

Prior to @dureuill's comment, this sentence read `Experimental features are not ready for production, […]`. We had more or less agreed on qualifying this statement by mentioning not all experimental features are ready for production.

If you still think it's problematic, I could add an extra sentence advising users to check a feature's dedicated page/discussion for more information.

I do think we need to keep some sort of warning, though. I understand the concern it might scare users away, but it seems misleading to say "experimental features are only experimental because the API may change" when that only applies to some of them.